pyproject.toml

```diff
@@ -106,3 +106,7 @@ initial-content = """
   __version__ = "0.0.0"
   __version_tuple__ = (0, 0, 0)
 """
+[tool.numpydoc_validation]
+checks = ["all", "EX01", "SA01", "ES01" ] # report on all checks, except the following
```

Comment:
Could you separate these out onto different lines and add a quick explanation of why we are ignoring them for quick reference?
```suggestion
checks = ["all", "EX01", "SA01", "ES01" ] # report on all checks, except the following
```